tests/conftest.py

```diff
@@ -316,7 +316,7 @@ def fxt_seg_data_entity() -> tuple[tuple, SegDataEntity, SegBatchDataEntity]:
     single_data_entity = SegDataEntity(
         image=fake_image,
         img_info=fake_image_info,
-        gt_seg_map=fake_masks,
+        masks=fake_masks,
```

Comment:
What is this change?

Reply:
I updated entities' key for mask in https://github.com/openvinotoolkit/training_extensions/pull/3724 to use the key seamlessly in transforms.
But that key in conftest was missed because this fixture is only called in mmseg test.